javascript/builtins/Date.json

```diff
@@ -2703,7 +2703,7 @@
                   "version_added": "24"
                 },
                 "chrome_android": {
-                  "version_added": null
+                  "version_added": "26"
```

Comment:
Did you mean to put 25 here?

Reply:
Not sure how it came out as "26" rather than "25"...